INSTALL.md

```diff
@@ -8,47 +8,72 @@ in their docs includes lots of very nice customisations for formatting documenta
 
 ## Building locally
 
+- `git clone --recurse-submodules git@github.com:opensafely/documentation.git`
+  - `--recurse-submodules` ensures you have the necessary copy of the `cohort-extractor` source
 - Create a virtual environment
 - `pip install -r requirements.txt`
 - `mkdocs serve`
```

Comment:
Is this whole section superseded by `just run`? and if not, it probably should be to match all our other infra?